cloudkeeper-one.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'settingslogic', '~> 2.0'
   spec.add_runtime_dependency 'activesupport', '~> 4.0'
   spec.add_runtime_dependency 'tilt', '~> 2.0'
+  spec.add_runtime_dependency 'opennebula', '~> 5.2'
```

Comment:
Recently, I started specifying dependencies like this:
```ruby
spec.add_runtime_dependency 'opennebula', '>= 5.2', '< 6'
```
What do you think? With conservative (`~> 5.2`) or open-ended (`>= 5.2`) dependencies, `gem build` will start complaining.